utam-core/src/main/java/utam/core/declarative/representation/MethodParameter.java

```diff
@@ -7,21 +7,52 @@
  */
 package utam.core.declarative.representation;
 
+import java.util.List;
+
 /**
- * method parameter
+ * method parameter, can be literal. Has type and name or value.
  *
  * @author elizaveta.ivanova
  * @since 226
  */
 public interface MethodParameter {
 
+  /**
+   * check if parameter is literal and can be skipped in method signature declaration
+   *
+   * @return true if passed by value AKA literal
+   */
   boolean isLiteral();
 
-  boolean isSelectorArgument();
```

Comment:
@jimevanssfdc it used to be only selector that would have nested args, now element as well. hence replaced it with new method getNestedParameters()